dodiscover/toporder/utils.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+def full_DAG(top_order: List[int]) -> NDArray:
```

Comment:
```suggestion
def full_dag(top_order: List[int]) -> NDArray:
```

Not a big fan of mixing upper-lower case unless it is a class